pom.xml

```diff
@@ -130,6 +130,21 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+              <groupId>org.jacoco</groupId>
+              <artifactId>jacoco-maven-plugin</artifactId>
+              <executions>
+                <execution>
+                  <id>prepare-agent-integration</id>
+                  <goals>
+                    <goal>prepare-agent-integration</goal>
+                  </goals>
+                  <configuration>
+                    <propertyName>coverage.command</propertyName>
+                  </configuration>
+                </execution>
+              </executions>
+            </plugin>
```

Comment:
Wouldn't this be better suited in the sling parent pom? We already configure Jacoco there in the `jacoco-report` profile.

Reply:
@rombert  yes that certainly would make sense - yet I didn't want to get into the game of porting the bundle to the latest (snapshot) parent as the game has changed since them (annotations etc.)

Reply:
@DominikSuess - then how about adding it to the parent pom in addition and creating a TODO - remove after updating to parent 35 ( or whichever version is coming next )?